semgrep-core/synthesizing/Synthesizer.ml

```diff
@@ -0,0 +1,14 @@
+open Common
```

Comment:
I usually try to also create a .mli where I put the interface (signature) of the exported functions.